extension/src/paymentHandler/make-lineitems-payment.handler.js

```diff
@@ -1,133 +0,0 @@
-import _ from 'lodash'
-import ctpClientBuilder from '../ctp.js'
-import makePaymentHandler from './make-payment.handler.js'
-import config from '../config/config.js'
-
-const ADYEN_PERCENTAGE_MINOR_UNIT = 10000
-const KLARNA_DEFAULT_LINE_ITEM_NAME = 'item'
-const KLARNA_DEFAULT_SHIPPING_METHOD_DESCRIPTION = 'shipping'
-
-async function execute(paymentObject) {
-  const makePaymentRequestObj = JSON.parse(
-    paymentObject.custom.fields.makePaymentRequest
-  )
-  const commercetoolsProjectKey =
-    paymentObject.custom.fields.commercetoolsProjectKey
-  if (!makePaymentRequestObj.lineItems) {
-    const ctpCart = await _fetchMatchingCart(
-      paymentObject,
-      commercetoolsProjectKey
-    )
-    if (ctpCart) {
-      makePaymentRequestObj.lineItems = createLineItems(paymentObject, ctpCart)
```

Comment:
Careful, since we are removing an existing convenience features like automatic lineItem generation out of the cart data here! I would probably reuse both files that you removed and did some adjustments/renaming since majority of the logic stays the same. Sessions and just different name for makePymentRequest/response.

Reply:
Added back these changes and since they are complicated to remove, I have created a follow up ticket to discuss with the team and migrate later.
https://jira.commercetools.com/browse/CTPI-761

Reply:
@praveenkumarct If we follow up this part as next step, I would suggest to keep this handler this moment, so that it helps us to understand how to migrate line-item features for web component v5.0

In addition, please keep `multitenancy.spec.js` and `payment-handler-authorization.spec.js`. They are required all the time no matter which web component version we are using.

Reply:
Yes, I have already reverted these changes back.